_posts/2020-10-20-SUGCH-2020-2.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: SUGCH 2020/2 - What's new in Sitecore 10?
+---
+
+This was the first event of our online Meetup series. We've adapted the meetup format a bit: To keep things brief, there will be only one presentation per event but we plan on having an event more often. The goal is once per month. Also, the event time was moved to office hours as this is more convenient for most.
+
+We were happy to have over 20 Sitecore enthusiasts attending this online meetup.
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">The SUGCH is starting. People are slowly showing up :) <a href="https://twitter.com/hashtag/sugch?src=hash&amp;ref_src=twsrc%5Etfw">#sugch</a> <a href="https://twitter.com/hashtag/sitecore?src=hash&amp;ref_src=twsrc%5Etfw">#sitecore</a> <a href="https://t.co/KyBtgXfnmn">pic.twitter.com/KyBtgXfnmn</a></p>&mdash; Tobias Studer (@studert) <a href="https://twitter.com/studert/status/1318552381693612032?ref_src=twsrc%5Etfw">October 20, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Presentations
+
+### What's new in Sitecore 10?
+
+We were lucky enough to have Jason St-Cyr, Technical Evangelist at Sitecore to show us around the brand new Sitecore 10 release. With a special focus on the developer features we've all ben waiting for. (Containerize all the things!)
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">21 people are listening to <a href="https://twitter.com/StCyrThoughts?ref_src=twsrc%5Etfw">@StCyrThoughts</a> talk about Sitecore 10 at <a href="https://twitter.com/hashtag/sugch?src=hash&amp;ref_src=twsrc%5Etfw">#sugch</a> - seeing all the new things in action <a href="https://twitter.com/hashtag/sitecore?src=hash&amp;ref_src=twsrc%5Etfw">#sitecore</a> <a href="https://twitter.com/hashtag/sitecore10?src=hash&amp;ref_src=twsrc%5Etfw">#sitecore10</a> <a href="https://t.co/WFr11SQLIY">pic.twitter.com/WFr11SQLIY</a></p>&mdash; Tobias Studer (@studert) <a href="https://twitter.com/studert/status/1318563139538006020?ref_src=twsrc%5Etfw">October 20, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Follow us on Twitter and visit the [moment on twitter](https://twitter.com/i/events/1241853914456240130) and see you at the next Sitecore User Group Switzerland.
```

Comment:
The moment on twitter link is showing the last onsite event. Did you create a new event and forgot to update the link or did you put this one intentionally?

Reply:
That one slipped in. Changed it to the twitter profile